### DISCUSSION TITLE
Adding a new compound to Database

### DISCUSSION THREAD

**Discussion by malangendorf** [reaktoro/reaktoro#268] [General]
Hello guys,

How it's going?

I'm using the reaktoro extension to DWSim software (Chemical Engineering Software) and I'll have to work with the ion chlorate (ClO3-) already present in the database.

But I need to calculate that dissociation from the sodium chlorate (NaClO3), that salt isn't present at the reaktoro database.

It's possible to add that salt to reaktoro database?

Thank you in advance,

Best Regards

**Comment by allanleal** [comment]
Hi @malangendorf , it is possible to add new substances to existing databases, by either editing the database file directly or by adding new `Species` objects to a `Database` object. 

The tutorial below should be helpful (it is about creating databases from scratch, but your use case is even simpler - just one extra species):
https://reaktoro.org/tutorials/advanced/creating-custom-databases.html

In Reaktoro v2, you can enter data for new substances using either standard thermodynamic properties of species or reactions (e.g., logK of reactions). The tutorial above covers this.

**Reply by malangendorf** [reply]
Thank you so much, @allanleal.
My first attempt I will try to make a reaction of dissociation on DWSIM (without a reaktoro reactor), I'm working with strong acids and salts. Maybe it works on my research.

But if I had some issues, I'll try to create a new compound based on your tutorial.

Thank you so much.

Best regards,

**Comment by allanleal** [comment]
Ok, let us know how it goes and if you need further assistance.
